src/DateTime.cc

```diff
@@ -527,11 +528,14 @@ DateTime DateTime::now(void) {
 void DateTime::initializeLeapSeconds(std::string const& leapString) {
     Leap l;
     leapSecTable.clear();
-    boost::regex re(
-            "^\\d{4}.*?=JD\\s*([\\d.]+)\\s+TAI-UTC=\\s+([\\d.]+)\\s+S"
-            " \\+ \\(MJD - ([\\d.]+)\\) X ([\\d.]+)\\s*S$");
-    for (boost::cregex_iterator i = make_regex_iterator(leapString.c_str(), re);
-         i != boost::cregex_iterator(); ++i) {
+    std::regex re(
+            "\\d{4}.*?=JD\\s*([\\d.]+)\\s+TAI-UTC=\\s+([\\d.]+)\\s+S"
+            " \\+ \\(MJD - ([\\d.]+)\\) X ([\\d.]+)\\s*S");
+
+    std::size_t sl = strlen(leapString.c_str());
```

Comment:
In this case, with pure ASCII, this should be the same as `leapString.length()`, right?

Overall, it might be better to replace all this conversion to C strings with an `sregex_iterator` and `stod` instead of `strtod`.